### PR TITLE
Fix #3269 When nacos.core.auth.enabled = true, user information is saved to the request in login

### LINKED ISSUE #3269: windows 下使用.\startup.cmd -m cluster -p embedded命令启动集群，启动后发现还是standalone mode
windows 下使用.\startup.cmd -m cluster -p embedded命令启动集群，启动后发现还是standalone mode

### PR DESCRIPTION
**Pull request by lidaoyang**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

NacosAuthManager request.setAttribute(user) and ConfigController getSrcUserName(req)

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [x] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: config/src/main/java/com/alibaba/nacos/config/server/controller/ConfigController.java [hunk 1]
```diff
@@ -87,31 +87,31 @@
 @RestController
 @RequestMapping(Constants.CONFIG_CONTROLLER_PATH)
 public class ConfigController {
-    
+
```
Comment (KomachiSion) [inline config/src/main/java/com/alibaba/nacos/config/server/controller/ConfigController.java:90]: Please do not change indent.
Please use nacos code style to reformat code.

see src/style/codeStyle.md

Comment (lidaoyang) [reply config/src/main/java/com/alibaba/nacos/config/server/controller/ConfigController.java]: ok

File: config/src/main/java/com/alibaba/nacos/config/server/service/repository/RowMapperManager.java [hunk 2]
```diff
@@ -47,138 +47,138 @@
  * @author <a href="mailto:liaochuntao@live.com">liaochuntao</a>
  */
 public final class RowMapperManager {
-    
+
```
Comment (KomachiSion) [inline config/src/main/java/com/alibaba/nacos/config/server/service/repository/RowMapperManager.java:50]: same question as above, do not change indent.

Comment (lidaoyang) [reply config/src/main/java/com/alibaba/nacos/config/server/service/repository/RowMapperManager.java:50]: ok

**Pull request merged**